azure-pipelines.release-fluentui.yml

```diff
@@ -115,7 +115,7 @@ jobs:
             echo "registry=http://registry.npmjs.org/" >> packages/fluentui/.npmrc
             echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> packages/fluentui/.npmrc
 
-            yarn release:fluentui:$(version) --yes
+            yarn release:fluentui:$(version) --yes &&
```

Comment:
If the intent is to exit on failure, we should probably specify `set -Eeuo pipefail` ([reference](https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/)) instead of littering `&&`

Reply:
We already do && in the other pipelines (it was just missing from this one). We'd definitely like a better solution, but I feel like having to add a `set` command before every single script is as bad or worse. So this is potentially something to investigate, but not in this PR.

Reply:
Reverted the && changes and made a separate PR #17459